h2o-py-mlflow-flavor/.gitignore

```diff
@@ -0,0 +1,44 @@
+# Backup files
```

Comment:
Do we need another .gitignore? A lot of staff is covered in h2o-3/.gitignore

Reply:
I removed the file, good catch